docs/sources/setup-grafana/configure-security/configure-authentication/saml-ui/index.md

```diff
@@ -0,0 +1,52 @@
+
+## Steps
+
+Follow these steps to configure and enable SAML integration:
+
+1. Sign in to Grafana and navigate to **Administration > Authentication > Configure SAML**.
+1. Complete the **General settings** section. Consult the following table for additional guidance about certain fields:
+
+| Field               | Description                                                                                                                                                                                                                                             |
+| ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| signup              | Determines whether to allow new Grafana user creation through SAML login. If disabled, then only existing Grafana users can log in with SAML.                                                                                                           |
+| single logout       | SAML's single logout feature allows users to log out from all applications associated with the current IdP session established via SAML SSO. Consult [SAML single logout documentation]]({{< relref "../saml/#single-logout" >}}) for more information. |
+| IdP initiated login | Allows users to log into Grafana directly from the SAML identity provider (IdP). Consult [IdP initiated login documentation]({{< relref "../saml/#idp-initiated-single-sign-on-sso" >}}) for more information.                                          |
+
+1. Click **Next: Key and certificate**.
+1. Provide a certificate and a private key that will be used by the service provider (Grafana) and the SAML identity provider. The private key needs to be issued in the [PKCS #8](https://en.wikipedia.org/wiki/PKCS_8) format. You can refer to our SAML documentation for [an example on how to generate SAML credentials]({{< relref "../saml/#example-of-how-to-generate-saml-credentials" >}}).
+1. Specify whether you want the outgoing requests to be signed, and, if so, which signature algorithm should be used.
+1. Click **Next: Connect Grafana with Identity Provider** and complete the section.
+1. Click **Next: User mapping**. If you wish to [map user information from SAML assertions]({{< relref "../saml/#assertion-mapping" >}}), complete this section. This section also allows you to set up SAML [team synchronization]({{< relref "../saml/#configure-team-sync" >}}), [role synchronization]({{< relref "../saml/#configure-role-sync" >}}) and [organization mapping]({{< relref "../saml/#configure-organization-mapping" >}}).
+1. Click **Next: Test and enable** and then click **Save and enable**.
```

Comment:
I would separate the test and enable from the Save. So something like:

1. Click **Next: Test and enable**.
    
    If there are issues...

1. Click Save and enable.

   Your SAML integration is now configured.

Reply:
Configuration errors are only highlighted once the user has tried to save the configuration. But we can talk through this once we look at the UI together.